compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -124,20 +134,37 @@ LogicalResult setMatmulLoweringConfig(IREE::GPU::TargetAttr target,
     return failure();
   }
 
-  // For now we are not being smart and trying to reshape dimensions to allow
-  // for better usage of intrinsics, and instead are tiling all dimensions
-  // except the inner most m, n, and k dimensions to 1.
```

Comment:
Are we doing this smart reshaping now?

Reply:
We are not doing smart reshaping, but we are taking advantage of all of the dimensions in another way by including them in the mma schedule heuristics. This is essentially unrolling along these outer dimensions.